_includes/graphql/customisation.md

```diff
@@ -0,0 +1,244 @@
+
+```typescript
+// The properties with ? are optional
+
+interface ParseGraphQLConfiguration {
+  // All classes enabled by default
+  // Provide an empty array to disable all classes
+  enabledForClasses?: Array<string>;
+
+  // Selectively disable specific classes
+  disabledForClasses?: Array<string>;
+
+  // Provide an array of per-class settings
+  classConfigs?: Array<{
+
+    // You must provide a className
+    // Only provide one config object per class
+    className: string;
+
+    type?: {
+
+      // By default, all fields can be sent for
+      // a create or update mutation. Use this
+      // setting to limit to specific fields.
+      inputFields?: {
+        create?: Array<string>;
+        update?: Array<string>;
+      };
+
+      // By default, all fields can be resolved
+      // on a get or find query. Use this to limit
+      // which fields can be selected.
+      outputFields?: Array<string>;
+
+      // By default, all valid fields can be used
+      // to filter a query. Use this to limit
+      // which fields can be used to constrain a query.
+      constraintFields?: Array<string>;
+
+      // By default, all valid fields can be used
+      // to sort the results of a query. Use this to
+      // limit which fields can be used to sort a query
+      // and which direction that sort can be set to.
+      sortFields?: {
+        field: string;
+        asc: boolean;
+        desc: boolean;
+      }[];
+    };
+
+    // By default, a get and find query type is created
+    // for all included classes. Use this to disable
+    // the available query types for this class.
+    query?: {
+      get?: boolean;
+      find?: boolean;
+    };
+
+    // By default, all write mutation types are 
+    // exposed for all included classes. Use this to disable
+    // the available mutation types for this class.
+    mutation?: {
+      create?: boolean;
+      update?: boolean;
+      destroy?: boolean;
+    };
+  }>
+}
+```
+
+### Set or Update Configuration
+
+We have provided a public API in ```ParseGraphQLServer``` which accepts the above JSON object for setting (and updating) your Parse GraphQL Configuration, ```setGraphQLConfig```:
+
+```typescript
+  const parseGraphQLServer = new ParseGraphQLServer(parseServer, {
+    graphQLPath: parseServerConfig.graphQLPath,
+    playgroundPath: parseServerConfig.playgroundPath
+  });
+
+  const config = {
+    // ... ParseGraphQLConfiguration
+  };
+
+  await parseGraphQLServer.setGraphQLConfig(config);       
+```
+
+### Include or Exclude Classes
+
+By default, all of your Parse classes, including the defaults such as ```Parse.User```, ```Parse.Session```, ```Parse.Role``` are added to the schema. You can restrict this using the ```enabledForClassess``` or ```disabledForClassess`` options, which accepts an array of Class names.
```

Comment:
```suggestion
By default, all of your Parse classes, including the defaults such as ```Parse.User```, ```Parse.Session```, ```Parse.Role``` are added to the schema. You can restrict this using the ```enabledForClassess``` or ```disabledForClassess`` options, which accepts an array of class names.
```